src/controller/CommissioningDelegate.h

```diff
@@ -457,6 +464,7 @@ class CommissioningParameters
     Optional<bool> mAttemptWiFiNetworkScan;
     Optional<bool> mAttemptThreadNetworkScan; // This automatically gets set to false when a ThreadOperationalDataset is set
     Optional<bool> mSkipCommissioningComplete;
+    Optional<bool> mNonConcurrentCommissioning;
```

Comment:
So here is a question: does this really need to be an Optional?  Or can it just be a boolean?

Reply:
Changed it to boolean